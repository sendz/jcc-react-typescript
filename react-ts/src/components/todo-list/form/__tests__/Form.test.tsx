import React from "react";
import {create} from "react-test-renderer";
import { ToDoListForm } from "..";

describe("snapshot test for Form", () => {
    test("render snapshot", () => {
        const tree = create(
            <ToDoListForm/>
        ).toJSON()
        expect(tree).toMatchSnapshot();
    });
});