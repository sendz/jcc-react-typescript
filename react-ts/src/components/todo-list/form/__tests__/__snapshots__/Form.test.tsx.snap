// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test for Form render snapshot 1`] = `
Array [
  <input
    data-testid="input-title"
    onChange={[Function]}
    placeholder="Title"
    type="text"
    value=""
  />,
  <button
    data-testid="save-button"
    onClick={[Function]}
  >
    Save
  </button>,
]
`;
